stylus-core/src/calls/context.rs

```diff
@@ -1,12 +1,8 @@
 // Copyright 2025-2026, Offchain Labs, Inc.
 // For licensing, see https://github.com/OffchainLabs/stylus-sdk-rs/blob/main/licenses/COPYRIGHT.md
 
-use crate::{
-    calls::{CallContext, MutatingCallContext, NonPayableCallContext, StaticCallContext},
-    storage::TopLevelStorage,
-};
+use crate::calls::{CallContext, MutatingCallContext, NonPayableCallContext, StaticCallContext};
```

Comment:
It is a bit confusing the Context traits being in the `mod.rs` and the Call struct being in `context.rs`. I would put everything in `mod.rs` and remove the `context.rs` file.